Mackbellemore/soen-390-team07
# chore(server): add inventory management for parts and materials

#

## Changes this PR introduces

- Creates default material lists required to build certain types of parts
- Validates stocks for material based on the above list



## How to test this PR

- create POST requests and PATCH requests for parts, note that now you need to pass a valid type to the part for a successful POST Request as well as having the needed materials in the DB. Valid types are:

  'handle_bar',
  'wheels',
  'chain',
  'frame',
  'pedal',
  'brakes',
  'seat',
  'fork',







## Notes for reviewers

- this is a note



## Checklist
